### PR TITLE
Add jammy content images to the build matrix

### PR DESCRIPTION
**Pull request by dbkegley**
- Modify maybe_install_quarto.sh to only install quarto for 1 version of bionic. 
- Install latest RC quarto (as of 2023-04-18) for _all_ jammy images.

### CODE REVIEW

File: content/base/Dockerfile.ubuntu2204 [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    && ./miniconda.sh -b -p /opt/miniconda \
+    && /opt/miniconda/bin/conda create --quiet --yes --prefix /opt/python/${PYTHON_VERSION} python=${PYTHON_VERSION} virtualenv \
+    && rm -f miniconda.sh \
+    # remove miniconda too, for size
+    && rm -rf /opt/miniconda
+
+# install quarto
+ARG QUARTO_VERSION=1.0.37
+COPY maybe_install_quarto.sh /tmp/maybe_install_quarto.sh
+RUN /opt/R/${R_VERSION}/bin/R -e 'install.packages("odbc", repos="https://packagemanager.rstudio.com/cran/__linux__/bionic/latest")' \
```
Comment (christierney) [inline content/base/Dockerfile.ubuntu2204:130]: should this be `jammy`?

File: content/base/Dockerfile.ubuntu2204 [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+RUN curl -fsSL -o miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh \
+    && chmod 755 miniconda.sh \
+    && ./miniconda.sh -b -p /opt/miniconda \
+    && /opt/miniconda/bin/conda create --quiet --yes --prefix /opt/python/${PYTHON_VERSION} python=${PYTHON_VERSION} virtualenv \
+    && rm -f miniconda.sh \
+    # remove miniconda too, for size
+    && rm -rf /opt/miniconda
+
+# install quarto
+ARG QUARTO_VERSION=1.0.37
```
Comment (christierney) [inline content/base/Dockerfile.ubuntu2204:128]: isn't this going to override the version in `maybe_install_quarto`?

File: content/base/Dockerfile.ubuntu2204 [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    && /opt/TinyTeX/bin/*/tlmgr path add \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install R -------------------------------------------------------------------#
+
+# Reference: https://docs.rstudio.com/resources/install-r/
+
+# We are NOT linking one of these R versions into the PATH.
+
+ARG R_DISTRIBUTION=ubuntu-1804
```
Comment (christierney) [inline content/base/Dockerfile.ubuntu2204:95]: 2204?

File: content/pro/Dockerfile.ubuntu2204 [hunk 4]
```diff
@@ -0,0 +1,20 @@
+ARG PYTHON_VERSION
+ARG R_VERSION
+FROM rstudio/content-base:r${R_VERSION}-py${PYTHON_VERSION}-bionic
```
Comment (christierney) [inline content/pro/Dockerfile.ubuntu2204:3]: `jammy`?

**Review by christierney — COMMENTED** [review]
quite possibly I'm just wrong about which versions/sources we need to use but I was surprised by the bionic references in the jammy Dockerfiles

File: content/pro/Dockerfile.ubuntu2204 (hunk 4 above)
Comment (dbkegley) [reply content/pro/Dockerfile.ubuntu2204:3]: This is what I get for copying everything verbatim

**Review by christierney — APPROVED** [review]

**Comment by ianpittwood** [comment]
I did make a PR for this same thing a while ago (it was probably missed since it's still in draft mode): #492 

I think we may want to use that one instead and reconcile the two. It has a few more changes to upgrade EoL'd Python and R versions and makes some more updates to make the `content` images to bring them up to speed with our other images.

**Comment by dbkegley** [comment]
@ianpittwood that sounds fine with me, is there anything we can do to help move your PR along? We would like to start using the jammy content images soon

**Comment by ianpittwood** [comment]
@dbkegley sure, I'm actually looking for some feedback on the R and Python versions used in the `matrix.json` file: https://github.com/rstudio/rstudio-docker-products/pull/492/files#diff-b96a499343e3b0a7c271edac8badf6d2d87ea589a25c08683536ee0296d165e8

I'm just looking for feedback on whether there's any version combinations we're missing and whether any of the ones removed are crucial in any way. I want to clean up the EoL'd versions to reduce build times and ensure we're using secure versions of R and Python, but I don't want to do that at anyone's expense.

I'll work on getting the `maybe_quarto_install.sh` script changes implemented in my PR and then once I get the build issues fixed I'll publish it for review.

**Pull request closed without merging**